test/integration/connect/envoy/case-cfg-splitter-cluster-peering/primary/verify.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "s1 proxy is running correct version" {
+  assert_envoy_version 19000
+}
+
+@test "s1 proxy admin is up" {
+  retry_default curl -f -s localhost:19000/stats -o /dev/null
+}
+
+@test "s2 proxy admin is up" {
+  retry_default curl -f -s localhost:19001/stats -o /dev/null
+}
+
+@test "s1 proxy listener should be up and have right cert" {
+  assert_proxy_presents_cert_uri localhost:21000 s1
+}
+
+@test "s2 proxies should be healthy in primary" {
+  assert_service_has_healthy_instances s2 1 primary
+}
+
+@test "s2 proxies should be healthy in alpha" {
+  assert_service_has_healthy_instances s2 1 alpha
+}
+
+@test "gateway-alpha should be up and listening" {
+  retry_long nc -z consul-alpha-client:4432
+}
+
+@test "peer the two clusters together" {
+  create_peering primary alpha
+}
+
+@test "s2 alpha proxies should be healthy in primary" {
+  assert_service_has_healthy_instances s2 1 primary "" "" primary-to-alpha
+}
+
+@test "s1 upstream should have healthy endpoints for s2 primary and alpha" {
+  assert_upstream_has_endpoints_in_status 127.0.0.1:19000 s2.default.primary.internal HEALTHY 1
+  assert_upstream_has_endpoints_in_status 127.0.0.1:19000 s2.default.primary-to-alpha.external HEALTHY 1
+}
+
+@test "s1 upstream should be split between peer and local dc" {
+  retry_long assert_url_header "http://127.0.0.1:5000/" "x-test-split" "primary"
+  [ "$status" -eq 0 ]
+  retry_long assert_url_header "http://127.0.0.1:5000/" "x-test-split" "alpha"
+  [ "$status" -eq 0 ]
```

Comment:
My understanding is that `retry_long` retries once per second for 30 seconds. This means that given a 50-50 splitter the chances of each case flaking is (1 / 2) ^ 30 or approximately 1 out of a billion.